src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -15,35 +15,101 @@
 // </copyright>
 
 using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 using System.Threading;
 using System.Threading.Tasks;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
 using OpenTelemetry.Trace.Export;
 
 namespace OpenTelemetry.Exporter.Console
 {
-    public class ConsoleExporter : SpanExporter
+    public class ConsoleExporter : ActivityExporter
     {
         private readonly JsonSerializerOptions serializerOptions;
+        private bool displayAsJson;
 
         public ConsoleExporter(ConsoleExporterOptions options)
         {
-            this.serializerOptions = new JsonSerializerOptions
+            this.serializerOptions = new JsonSerializerOptions()
             {
-                WriteIndented = options.Pretty,
+                WriteIndented = true,
             };
 
+            this.displayAsJson = options.DisplayAsJson;
+
             this.serializerOptions.Converters.Add(new JsonStringEnumConverter());
             this.serializerOptions.Converters.Add(new ActivitySpanIdConverter());
             this.serializerOptions.Converters.Add(new ActivityTraceIdConverter());
         }
 
-        public override Task<ExportResult> ExportAsync(IEnumerable<SpanData> batch, CancellationToken cancellationToken)
+        public override Task<ExportResult> ExportAsync(IEnumerable<Activity> activityBatch, CancellationToken cancellationToken)
         {
-            foreach (var span in batch)
+            foreach (var activity in activityBatch)
             {
-                System.Console.WriteLine(JsonSerializer.Serialize(span, this.serializerOptions));
+                if (this.displayAsJson)
+                {
+                    System.Console.WriteLine(JsonSerializer.Serialize(activity, this.serializerOptions));
+                }
+                else
+                {
+                    System.Console.WriteLine("Activity ID - " + activity.Id);
+                    if (!string.IsNullOrEmpty(activity.ParentId))
+                    {
+                        System.Console.WriteLine("Activity ParentId - " + activity.ParentId);
+                    }
+
+                    System.Console.WriteLine("Activity OperationName - " + activity.OperationName);
```

Comment:
Curious, do we expect synchronization (e.g. having the entire `else` block running mutual exclusively)?

Reply:
Not sure I understood the question. Can you clarify

Reply:
If multiple threads are triggering this piece of code (e.g. using simple exporter/processor), do we expect the console output to be mixed up?

Reply:
Got it. Yes i think it'll mix up the outputs. Its okay (?) as console is just for learning how activity looks like and we can keep it very simple.